package-lock.json

```diff
@@ -1,8 +1,12252 @@
 {
   "name": "@hulu/roca",
   "version": "0.21.0",
-  "lockfileVersion": 1,
```

Comment:
Careful, we don't want to upgrade our lockfile version as part of this PR. We could do it separately (or better yet, switch to Yarn), but we should keep it as-is for this PR.